Lib/python/pyhead.swg

```diff
@@ -1,12 +1,9 @@
-#if PY_VERSION_HEX >= 0x03030000 && !defined(SWIG_NO_HEAPTYPES)
+#if !defined(SWIG_NO_HEAPTYPES)
 #if !defined(SWIG_HEAPTYPES)
 #define SWIG_HEAPTYPES
 #endif
 #endif
 
-/* Compatibility macros for Python 3 */
-#if PY_VERSION_HEX >= 0x03000000
-
```

Comment:
If we're keeping these macros (which probably makes sense as user typemaps may be relying on them) then we should keep the comment saying what they're for.